bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -103,7 +105,9 @@ trait CachedBackend extends VoTableBackend {
       def go(pos: Int):Option[(Int, QueryResult)] =
         a.lift(pos) match {
           case None                                                            => None
-          case Some(CacheEntry(SearchKey(query:ConeSearchCatalogQuery, r), v)) => if (query.isSuperSetOf(k.query)) Some((pos, v)) else go(pos + 1)
+          case Some(CacheEntry(SearchKey(query:ConeSearchCatalogQuery, _), v)) =>
+            // Note we need to compare against the widened query
+            if (widen(query).isSuperSetOf(k.query)) Some((pos, v)) else go(pos + 1)
           case _                                                               => None // Not caching named queries so far
```

Comment:
I suppose you could collapse the two branches that return None.  In fact, maybe you can

````scala
a.lift(pos).collect {
  case Some(CacheEntry ...) => ..
}
````